### DISCUSSION TITLE
Date from MSSQL showing as previous day

### DISCUSSION THREAD

**Discussion by jdids247** [Budibase/budibase#10060] [Help]
We have an app built on an existing table in MSSQL. 

The underlying table data is stored correctly but displaying as a day behind.

The date format is YYYY-MM-DD

This issue is almost exact to what we're experiencing (https://github.com/Budibase/budibase/issues/7531) but we don't have the time zone option since we created the table outside of the Budibase UI

Do we need to delete those date fields, then re-add them using the Budibase UI to display those dates correctly?

**Comment by melohagan** [comment]
Hey @jdids247 

So the *Ignore timezone* flag is on in your case? You can edit the columns of the tables in Budibase.

**Reply by jdids247** [reply]
That isn't an option. I had to write a javascript function to get them to show correctly.

![image](https://user-images.githubusercontent.com/110419431/229520942-cec3aacb-279c-4c07-ad69-f007a5ade01a.png)

**Reply by melohagan** [reply]
Hmm looking at the codebase I can see this line: `{#if datasource?.source !== "ORACLE" && datasource?.source !== "SQL_SERVER"}`

@aptkingston Might have the answer for this one!

**Comment by Reading90** [comment]
I have the same issue, yes. I'm using Postgres as the database source and am self-hosting it on Docker. When testing the app I made, filling out a form and submitting it makes the date go back by one day (ex: entered in as 04/24/2023, shows up in the table as 04/23/2023). I can fix this by editing the row - both on the published app side and on the backend - and the corrected date saves fine.

Date format is MM/DD/YYYY. No time is entered when putting the date in (times are off in the UI).

I put ignore time zones on (not in the database but in the Budibase UI) just in case that was the issue, but it didn't change anything. Would appreciate any ideas, thanks!

**Reply by melohagan** [reply]
Hey @Reading90 that may possibly be a bug. It could be worth creating a bug issue with the reproduction steps and screenshots!

**Reply by Reading90** [reply]
Ok thanks! I'll go do that
